grimshot

```diff
@@ -78,7 +78,16 @@ if [ "$ACTION" != "save" ] && [ "$ACTION" != "copy" ] && [ "$ACTION" != "savecop
 fi
 
 notify() {
-  notify-send -t 3000 -a grimshot "$@"
+  if [ "$ACTION" = "copy" ]; then
+    notify-send -t 3000 -a grimshot "$@"
+  else
+    action=$(notify-send -t 3000 -a grimshot --action=file='Open File' --action=dir='Open Directory' "$@")
+    if [ "$action" = "file" ]; then
+      xdg-open "$FILE"
+    elif [ "$action" = "dir" ]; then
+      xdg-open "$(dirname "$FILE")"
+    fi
```

Comment:
```suggestion
    if [ "$action" = "dir" ]; then
      xdg-open "$(dirname "$FILE")"
    else
      xdg-open "$FILE"
    fi
```
With notification programs like `mako`, you can't select the action. So make opening file as default would be better.

Reply:
Oh this suggestion makes it always open the file, ignore that.

Reply:
Do you have an idea for this?